### PR TITLE
use nodemon to hot-reload

### LINKED ISSUE #26: A hot reloading in `api:dev` script
It's a pain having to restart the API every time a slight tweak is made. It should reload automatically when saving a file in the api folder.

### PR DESCRIPTION
**Pull request by nmcc1212**
fixes #26 
uses nodemon to watch api folder as asked

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,19 +12,43 @@
...
+    {
+      "name": "Srijan Vaddadi",
+      "email": "srijan.vaddadi@gmail.com",
+      "url": "https://srijan.uk"
+    },
+    {
+      "name": "sanju",
+      "email": "ghandhisanju@gmail.com",
+      "url": "https://n1al.com"
+    }
```
Comment (fergcb) [inline package.json:51]: Why have the contributors been formatted like this?

**Review by fergcb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nmcc1212) [reply package.json:51]: 🤷‍♂️

**Review by SSG27 — DISMISSED** [review]
reformatteed contributors, but looks good so idc :D

**Review by SSG27 — APPROVED** [review]
:( liked the. old contributors more.

**Review by fergcb — APPROVED** [review]

**Pull request merged**